sbin/pfdetect

```diff
@@ -156,10 +156,11 @@ sub _run_detector {
         my $line = $_;
         $logger->info("alert received: '$line'");
         my $data = $parser->parse($line);
-        unless($data) {
+        unless($data || $data eq 0) {
```

Comment:
I said that because Zammit said :
`Les == c'est pour les PDs`

So I basically had no choice since I'm part of ZammitCorp to be on Zammits side.

Last time I listen to you sir....